Packs/AzureFirewall/Integrations/AzureFirewall/AzureFirewall_description.md

```diff
@@ -34,20 +70,10 @@ Follow one of these steps for authentication based on Azure Managed Identities:
 
 For information about Azure Managed Identities see [Managed identities for Azure resources](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview)
 
+
 ## Get Subscription ID and the Resource Group Values
```

Comment:
Its not clear why we are doing this. Is it to map to XSOAR fields? We should say.